docs/admin/deployment.md

```diff
@@ -1015,64 +1015,67 @@ the necessary DAOS metadata indicating that the server has been formatted.
 When starting, `daos_server` will skip `maintenance mode` and attempt to start
 I/O engines if valid DAOS metadata is found in `scm_mount`.
 
+
 ## Agent Setup
 
-This section addresses how to configure the DAOS agents on the storage
-nodes before starting it.
+This section addresses how to configure the DAOS Agents on the client nodes
+(and optionally on the DAOS server nodes).
```

Comment:
I'd lean toward removing this line. I think it adds confusion. The daos_server processes don't use the agent at all-- if it is installed on a server node with no client processes, it doesn't actually do anything.

Reply:
i'm usually configuring the agent on the servers too, because i want the dmg command on the servers and dmg is packaged in the same daos-client RPM as the agent. and while dmg doesn't use the agent, i prefer to configure it when it gets installed. but agreed that it adds confusion here, so best to remove that paranthesis.